build/docker/minio/test-import.ndjson

```diff
@@ -0,0 +1,3 @@
+{"resourceType":"Patient","id":"72b0d93c-93d0-43d9-94a8-d5154ce07152","meta":{"versionId":"1","lastUpdated":"2020-04-17T00:09:36.910431Z"},"name":[{"family":"Doe","given":["import-1"]}],"telecom":[{"system":"phone","value":"555-1122","use":"home"}],"birthDate":"1970-01-01"}
+{"resourceType":"Patient","id":"72b0d93c-93d0-43d9-94a8-d5154ce07153","meta":{"versionId":"1","lastUpdated":"2020-04-17T00:09:36.910431Z"},"name":[{"family":"Doe","given":["import-2"]}],"telecom":[{"system":"phone","value":"555-1122","use":"home"}],"birthDate":"1970-01-01"}
+{"resourceType":"Patient","id":"72b0d93c-93d0-43d9-94a8-d5154ce07154","meta":{"versionId":"1","lastUpdated":"2020-04-17T00:09:36.910431Z"},"name":[{"family":"Doe","given":["import-3"]}],"telecom":[{"system":"phone","value":"555-1122","use":"home"}],"birthDate":"1970-01-01"}
```

Comment:
Shouldn't we remove the test data in /fhir-server-test/src/test/resources/testdata/import-operation/test-import.ndjson ? or copy from there to here.  Then there will be one place to update.

Reply:
ha, good point. I can reused it!